<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureControlWidget</class>
 <widget class="QWidget" name="CaptureControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="channelIdentifierLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startStopButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>64</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QFrame" name="recordStatusFrame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>65</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="recordStatusLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recording</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="extraInformationLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>4</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>64</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="CommonWidgets.qrc">
     <normaloff>:/icons/Resources/icon_Save.png</normaloff>:/icons/Resources/icon_Save.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="requestedFrameRateSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="requestedFrameRateLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>34</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Requested Frame Rate:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>230</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>channelIdentifierLabel</zorder>
  <zorder>startStopButton</zorder>
  <zorder>recordStatusFrame</zorder>
  <zorder>extraInformationLabel</zorder>
  <zorder>saveButton</zorder>
  <zorder>requestedFrameRateSpinBox</zorder>
  <zorder>requestedFrameRateLabel</zorder>
 </widget>
 <resources>
  <include location="CommonWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
